x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvSumTests.java

```diff
@@ -65,7 +65,7 @@ public static Iterable<Object[]> parameters() {
             data.add(asLongUnsigned(UNSIGNED_LONG_MAX));
             return data;
         }));
-        return parameterSuppliersFromTypedData(cases);
+        return parameterSuppliersFromTypedData(anyNullIsNull(false, cases));
```

Comment:
Can we `parameterSuppliersFromTypedDataWithDefaultChecks`?

Same for other tests, if possible

Reply:
I did it where it passed easily but didn't want to get stuck on it in the places where it didn't. At least, that's what I think I did. I think we should do the same thing as this but for `errorsForCasesWithoutExamples` and see what's left. Those can take a lot longer though.

Reply:
> I did it where it passed easily

Say no more! We can continue later indeed, thanks!